### PR TITLE
Fix out of date code

### PR DESCRIPTION
**Pull request by tgriffith-gpsw**
Update the Python code to handle the docker environment correctly.

### CODE REVIEW

File: python/helper.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 
 
 def getPayload():
     file = None
-    for i in range(len(sys.argv)):
-        if sys.argv[i] == '-payload' and (i + 1) < len(sys.argv):
-            file = sys.argv[i + 1]
-            break
+    
+    if os.environ.has_key('PAYLOAD_FILE'):
```
Comment (ulandj) [inline python/helper.py]: `has_key()` is deprecated. use the `in` operator instead. E.g.

``` python
if 'PAYLOAD_FILE' in os.environ:
```

**Comment by carimura** [comment]
Hey Tague, thanks for the PR. @ulandj look good now?

**Comment by ulandj** [comment]
@carimura  need to replace `has_key` to `in` operator... everything else looks good

File: python/helper.py [hunk 2]
```diff
@@ -16,9 +20,13 @@ def getPayload():
 
 def getTaskId():
     task_id = None
-    for i in range(len(sys.argv)):
-        if sys.argv[i] == '-id' and (i + 1) < len(sys.argv):
-            task_id = sys.argv[i + 1]
-            break
+    
+    if os.environ.has_key('TASK_ID'):
```
Comment (ulandj) [inline python/helper.py]: and here need to replace `has_key`

**Comment by tgriffith-gpsw** [comment]
@ulandj updated.

**Comment by ulandj** [comment]
Thanks

**Pull request merged**